From: Felipe Sateler <fsateler@debian.org>
Date: Sun, 14 Apr 2019 12:44:19 -0400
Subject: Don't remove symbol table and relocation information from binaries

This can be done post install by separate tools, and prevents building separate
debug symbol packages
---
 src/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index bcd8966..719431f 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -45,7 +45,6 @@ CXX		?=	c++
 		$(LIBRARIES)						\
 		-fvisibility=hidden					\
 		-fvisibility-inlines-hidden				\
-		-s							\
 		-Wl,--version-script=gcc_exports.map			
 
 ../plugins/%.so:	plugins/%.cpp ladspa.h gcc_exports.map
@@ -59,7 +58,6 @@ CXX		?=	c++
 		$(LIBRARIES)						\
 		-fvisibility=hidden					\
 		-fvisibility-inlines-hidden				\
-		-s							\
 		-Wl,--version-script=gcc_exports.map			
 
 ###############################################################################
